docs/12-ContinuousIntegration.md

```diff
@@ -119,8 +119,28 @@ Travis doesn't provide visualization for XML or HTML reports so you can't view r
 
 ## GitLab
 
-[Work In Progress]
+If a file ```.gitlab-ci.yml``` exists in the root of the git repository, gitlab will run a pipeline each time you push to the gitlab server. The file configures the docker image that will be called. Below is a sample which loads a php7 docker image, clones your files, installs composer dependencies, runs the built-in php webserver and finally runs codeception:
+
+```
+image: mychiara/php7
```

Comment:
yes, I still prefer official because if this goes to official documentation this image should be reliable. I know nothing about its maintaining policy and I don't want to receive issues if something goes wrong with it. We just need to understand that people will do copy-pasting this configs today, tomorrow, and years after. And we need to be sure that this will work anyway.

That's why I prefer official. Also, didn't you check `codeception/codeception` image? It has composer built in as well